_posts/2024-07-12-ecrs-summary.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: single
+title:  "A Considerably Constructive Coral Conference: ECRS 2024"
+excerpt: "Presenting early-stage PhD work at Europe's largest coral conference"
+categories: conference
+toc: false
+
```

Comment:
Add `author: Orlando Timmerman` here to make your profile appear in the sidebar.